modules/core/src/controllers/controller.js

```diff
@@ -21,6 +21,11 @@
 /* eslint-disable max-statements, complexity */
 import TransitionManager from './transition-manager';
 
+/**
+ *
+ * @typedef { import('hammerjs')}
```

Comment:
We are not using JSDoc to define types. All type definitions should go into .d.ts files.